react-frontend/src/components/DigitalFramework/digitaltable.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { Col, Row, Grid } from 'react-flexbox-grid';
+
+function DigitalTable(props) {
+  const initial = window.innerWidth < 800;
+  const [isMobile, setIsMobile] = useState(initial);
```

Comment:
can we accomplish mobile responsiveness with css and media queries?

Reply:
There are titles that render in the component conditionally, we could just put visible / not visible in the css, at the addition of more classes